wrappers/wrapper.rb

```diff
@@ -478,6 +478,7 @@ def add_unassigned(unfeasible, vrp, service, reason)
 
             add_unassigned(unfeasible, vrp, service_in, "In a relation with an unfeasible service: #{service.id}")
           }
+          vrp.relations.delete(relation)
```

Comment:
Why do we delete a relation if there is an infeasible service in it? There might be other feasible services in it, no?

Reply:
:face_exhaling: :facepalm: ALL_OR_NONE_RELATIONS

Reply:
we are in the context of a `ALL_OR_NONE_RELATIONS` so it requires to delete all the services and the relation as well otherwise we have a relation without the corresponding services in the problem